<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="UBL-LC-DINESH" failures="1" tests="7" name="home" time="0.077" errors="0" timestamp="2023-06-29T14:17:20 IST">
  <testcase classname="com.selenium.TestNG.day3" name="weblogindata" time="0.001"/>
  <testcase classname="com.selenium.TestNG.day3" name="weblogindata" time="0.002"/>
  <testcase classname="com.selenium.TestNG.day3" name="websignup" time="0.001"/>
  <testcase classname="com.selenium.TestNG.day3" name="webtimeout" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.selenium.TestNG.day3.webtimeout(day3.java:22)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- webtimeout -->
  <testcase classname="com.selenium.TestNG.day3" name="websignin" time="0.001"/>
  <testcase classname="com.selenium.TestNG.day2" name="homesignin" time="0.003"/>
  <testcase classname="com.selenium.TestNG.day2" name="homesignout" time="0.001"/>
  <testcase classname="com.selenium.TestNG.day2" name="homesignup" time="0.0">
    <ignored/>
  </testcase> <!-- homesignup -->
  <testcase classname="com.selenium.TestNG.day3" name="websignout" time="0.0">
    <ignored/>
  </testcase> <!-- websignout -->
</testsuite> <!-- home -->
